testnets/cosmosicsprovidertestnet/chain.json

```diff
@@ -33,22 +33,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/cosmos/gaia",
-    "recommended_version": "v20.0.0-rc0",
+    "recommended_version": "v20.0.0",
     "compatible_versions": [
-      "v20.0.0-rc0"
+      "v20.0.0"
     ],
     "cosmos_sdk_version": "v0.50.9-lsm",
     "consensus": {
       "type": "cometbft",
       "version": "v0.38.11"
     },
     "binaries": {
-      "linux/amd64": "https://github.com/cosmos/gaia/releases/download/v20.0.0-rc0/gaiad-v20.0.0-rc0-linux-amd64",
-      "linux/arm64": "https://github.com/cosmos/gaia/releases/download/v20.0.0-rc0/gaiad-v20.0.0-rc0-linux-arm64",
-      "darwin/amd64": "https://github.com/cosmos/gaia/releases/download/v20.0.0-rc0/gaiad-v20.0.0-rc0-darwin-amd64",
-      "darwin/arm64": "https://github.com/cosmos/gaia/releases/download/v20.0.0-rc0/gaiad-v20.0.0-rc0-darwin-arm64",
-      "windows/amd64": "https://github.com/cosmos/gaia/releases/download/v20.0.0-rc0/gaiad-v20.0.0-rc0-darwin-amd64",
-      "windows/arm64": "https://github.com/cosmos/gaia/releases/download/v20.0.0-rc0/gaiad-v20.0.0-rc0-windows-arm64.exe"
+      "linux/amd64": "https://github.com/cosmos/gaia/releases/download/v20.0.0/gaiad-v20.0.0-linux-amd64",
```

Comment:
Please update the version object within the versions array in the same way (both: binaries, and the compatible versions (should it include both v20.0.0 and v20.0.0.0-rc0?))

Reply:
Thanks! I updated the version object so the binaries and compatible versions match.